extensions/sql-migration/src/wizard/loginMigrationStatusPage.ts

```diff
@@ -458,7 +458,7 @@ export class LoginMigrationStatusPage extends MigrationWizardPage {
 			{
 				...getTelemetryProps(this.migrationStateModel),
 				'loginsAuthType': this.migrationStateModel._loginMigrationModel.loginsAuthType,
-				'numberLoginsFailingPerStep': JSON.stringify(Array.from(this.migrationStateModel._loginMigrationModel.errorCountMap)),
+				'numberLoginsFailingPerStep': this.migrationStateModel._loginMigrationModel.errorCountMapString.toString(),
```

Comment:
In this case errorCountMapString is a string[], does toString() return what you want?

Reply:
yes it returns in the needed format [MigrateLogins:{}],[EstablishUserMapping:{"DatabaseNotFoundOnDestinationException":1}],[MigrateServerRolesAndSetPermissions:{"RoleAlreadyExistsException":1}]